.taskcluster.yml

```diff
@@ -341,6 +341,10 @@ tasks:
                    git config --global user.name moz.tools Bot &&
                    git clone --quiet ${repository} &&
                    cd rust-code-analysis &&
+                   cd tree-sitter-ccomment && taskboot cargo-publish && cd .. &&
+                   cd tree-sitter-preproc && taskboot cargo-publish && cd .. &&
+                   cd tree-sitter-mozjs && taskboot cargo-publish && cd .. &&
+                   cd tree-sitter-mozcpp && taskboot cargo-publish && cd .. &&
```

Comment:
What happens if the release already exists on crates.io?

Reply:
`taskboot cargo-publish` does not discriminate among errors, so if a package is already published, it returns an error and the CI is interrupted

Reply:
OK, so this way we need to bump the versions of those crates even if nothing changed within them.

Reply:
WFM